src/lib/filter.js

```diff
@@ -26,7 +28,6 @@ export default function filter({
                 return filterSelectedItems ? x[itemId] === item[itemId] : false;
```

Comment:
This can be simplified 

Current:
```
 matchesFilter = !value.some((x) => {
                return filterSelectedItems ? x[itemId] === item[itemId] : false;
            });
```

To: 
` matchesFilter = !value.some((x) => filterSelectedItems && x[itemId] === item[itemId]);`